contrib/packaging_python/mk_clean_wheel.sh

```diff
@@ -26,6 +26,7 @@
 PYTHONBIN=$1
 CONFIG="$2"
 PYVERSION=$($PYTHONBIN -c "import sys; print(sys.implementation.name + sys.version.split()[0])")
+PYINCLUDE=$($PYTHONBIN -c "import sysconfig; print(sysconfig.get_paths()['include'])")
```

Comment:
Needed because of scikit, otherwise it will include files from another python version installed in the system.

Reply:
Add the above comment to the bash script itself.